pci/src/vfio.rs

```diff
@@ -375,6 +376,19 @@ impl VfioCommon {
             let region_size: u64;
             let bar_addr: GuestAddress;
 
+            let base_addr = if let Some(resources) = &resources {
```

Comment:
I think `base_addr` could have a better name e.g. `restored_bar_addr`

Reply:
Ok I'll change it.